### DISCUSSION TITLE
我自定义一个 这个 壳子，怎么将壳子里面的数据传递给 拖入的组件里

### DISCUSSION THREAD

**Discussion by w990752800** [opentiny/tiny-engine#384] [Q&A]
<img width="326" alt="image" src="https://github.com/opentiny/tiny-engine/assets/23731304/4af6aa23-6f04-45ef-88f8-97129a3b49de">

**Comment by chilingling** [comment]
https://opentiny.design/tiny-engine#/help-center/course/engine/9272323322804964

可以看“如何使用插槽“一节
@w990752800

**Reply by w990752800** [reply]
我使用的本地的mockServer，不能自定义区块。。。有没有其他方法

**Reply by chilingling** [reply]
WsBlock 已经是自定义区块，你直接绑定变量 message 出码应该就可以生效
@w990752800

**Reply by w990752800** [reply]
直接在拖入的组件中这样使用？ 
<img width="690" alt="image" src="https://github.com/opentiny/tiny-engine/assets/23731304/5184c43c-a852-4eeb-a9d5-9b55a2126a3c">

**Reply by chilingling** [reply]
对的，你试试 @w990752800

**Reply by w990752800** [reply]
<img width="1136" alt="image" src="https://github.com/opentiny/tiny-engine/assets/23731304/6d9452bc-c5f0-4000-95b5-290648038899">
不行，显示不了我上面定义的message

**Reply by chilingling** [reply]
WsBlock 需要 schema 声明插槽
![image](https://github.com/opentiny/tiny-engine/assets/26962197/8125182e-51a8-4e7d-a264-9b82d13ea027)
